lib/rules/disallow-capitalized-comments.js

```diff
@@ -57,7 +57,7 @@ module.exports.prototype = {
         var letterPattern = require('../../patterns/L');
         var lowerCasePattern = require('../../patterns/Ll');
 
-        file.getComments().forEach(function(comment) {
+        file.iterateTokensByType(['Line', 'Block'], function(comment) {
```

Comment:
I think that a rename change to how these tokens are typed should be a separate standalone PR.